syngenta_digital_dta/s3/adapter.py

```diff
@@ -104,6 +104,15 @@ def create_presigned_post_url(self, **kwargs):
         )
         return results
 
+    def is_exist(self, s3_path):
```

Comment:
```suggestion
    def object_exist(self, **kwargs):
```

Think is a more accurate method name. The other method could apply to any resource within S3 instead of what you are implying, the object. 

Also to stay consistent with the other methods we should use kwargs. This way it's easy to add more non, breaking features in the future. 

I think your tests can remain the same since python 3 will interpret positional arguments with the same variable name as a keyword argument. Just need to change the inner logic of the method to reference the kwarg.

Reply:
Done .. I just wonder why tests are failing .. also unauthorized status when I try to access pipeline